ai-backend/scripts/retrieve.py

```diff
@@ -25,18 +93,57 @@
     {
         "bool": {
             "should": [
-                {"term": {"metadata.embedding_type": "title"}},
+                {"term": {"metadata.embedding_type": "fulltext"}},
                 {"term": {"metadata.embedding_type": "abstract"}},
             ],
             "minimum_should_match": 1,
         }
     }
 ]
-docs = vector_store.similarity_search_with_score(
-    "What is a star?", k=100, filter=search_filter
+
+search_query = """
+Whats being discussed in the Collider constraints?
+"""
+
+prompt = PromptTemplate.from_template("""
+    You are an assistant for question-answering tasks in High Enegery Physics.
```

Comment:
Please fetch the prompts from LangFuse to be able to test more easily. Also, small typo in `Enegery`.